pagopa/pn-frontend
# Feat/pn 2394 separare pagine informativa

## Short description
Both in pn-pa-webapp and pn-personafisica-webapp, the terms of service and privacy policy are now in two separate pages

## List of changes proposed in this pull request
- The terms of service and privacy policy are now in two separate pages, TermsOfService.page.tsx and PrivacyPolicy.page.tsx respectively
- Renamed old TermsOfService page to ToSAcceptance page

## How to test
- Check that terms of service and privacy policy are on 2 different routes and pages('informativa-privacy', 'termini di servizio') in pn-pa-webapp and pn-personafisica-webapp
- Check that links in the footer point to the respective pages.
